app/assets/stylesheets/pq.scss

```diff
@@ -753,6 +762,15 @@ textarea.form-control {
     #progress-panel {
       margin: 0px;
       padding: 0px;
+
+      #choices {
+       background-color: $light-blue;
+       color: $page-colour;
```

Comment:
there's already a class with the same attributes for buttons. It can be re-used.